doc/architecture/decisions/0011-build-hcadmin-app-with-qt-c.md

```diff
@@ -4,7 +4,7 @@ Date: 2018-05-16
 
 ## Status
 
-proposed
```

Comment:
I would say we should actually mark this as "obsoleted" by the Container ADR, which is a generalization of this ADR from an "architecture" point of view.  It doesn't mean we don't build holosqape, it just means that architecturally we now recognize that there are many ways to "admin" i.e. deploy your holochain instances which depend on the container type and the use-case type.